### ISSUE TITLE
Device panic after userspace reboot

### ISSUE THREAD

**Issue by Banaantje04** [checkra1n/BugTracker#2025]
**Tell us about your setup:**
1. What iDevice are you using?
iPad 5 (WiFi)
2. On what version of iOS is it?
iOS 14.1
3. What version of checkra1n are you using?
0.12.2
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Linux Ubuntu 
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB-A from Apple

**What are the steps to reproduce the issue?**
1. I was looking at the iPad storage tab in Settings
2. Open terminal
3. enter command `launchctl reboot userspace`
...

**What do you expect, and what is happening instead?**
An userspace reboot.
The screen froze, after a while it showed a loading circle. Then an apple logo.

**Did you see a popup on the device stating it entered "Safe Mode" due to an error?**
No

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
I don’t know

**Any other info, error logs, screenshots, ...?**
Panic log: (extension changed so that github will allow it)
[panic-full-2021-03-05-120925.000.log](https://github.com/checkra1n/BugTracker/files/6092742/panic-full-2021-03-05-120925.000.log)

**Comment by Siguza** [comment]
Could you see if you can still reproduce this under 0.12.3?  
I tried that on a bunch of devices, and it always seemed to work fine.

**Comment by Banaantje04** [comment]
I could try that yeah. I have discovered some more information on it since I first reported it, I honestly forgot I made this. The problem seems to be that sometimes processes don't spawn anymore, the only thing that pops up are a couple of xpcproxies. And because no new processes spawn userspace reboots and ldrestarts just make the kernel panic after daemons time out. I still have no idea how to reproduce it though, after I made that report I had it happen to me almost daily for two weeks, after that it only happened on the 14th of April. It happens on both check/odysseyra1n and Taurine so I am not actually sure anymore if it's checkra1n specific.
